core/nodejsActionBase/runner.js

```diff
@@ -21,6 +21,9 @@
  * This file (runner.js) must currently live in root directory for nodeJsAction.
  */
 var util = require('util');
+var child_process = require('child_process');
+var fs = require('fs');
+var path = require('path');
```

Comment:
I believe what you are saying is correct. My gut feeling is that in all cases, we should delay this loading until we know we need it. If that initialization happens to get re-used by a warm container, then that is even better.